src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -2629,7 +2669,15 @@ SpellMissInfo Unit::MagicSpellHitResult(Unit* victim, SpellInfo const* spellInfo
         }
 
         if (hasAura)
-            tmp += victim->GetTotalAuraModifierByMiscValue(SPELL_AURA_MOD_DEBUFF_RESISTANCE, static_cast<int32>(spellInfo->Dispel)) * 100;
```

Comment:
Why is this needed? TotalAuraModifier will automatically add all effects (possitive and negative)

Not to mention, that there is only one such aura in 3.3.5: 53659 - Sacred Cleansing